client/verta/tests/registry/model_version/test_from_run.py

```diff
@@ -37,29 +37,4 @@ def test_from_run(self, experiment_run, model_for_deployment, registered_model):
             )
             assert np.array_equal(model_version.get_artifact("some-artifact"), artifact)
 
-    def test_from_run_diff_workspaces(
-        self, client, experiment_run, organization, created_entities
-    ):
-        registered_model = client.create_registered_model(workspace=organization.name)
-        created_entities.append(registered_model)
-
-        model_version = registered_model.create_version_from_run(
-            run_id=experiment_run.id, name="From Run {}".format(experiment_run.id)
-        )
-
-        assert model_version.workspace != experiment_run.workspace
-
-    def test_from_run_diff_workspaces_no_access_error(
-        self, experiment_run, client_2, created_entities
-    ):
-        registered_model = client_2.create_registered_model()
-        created_entities.append(registered_model)
-
-        with pytest.raises(requests.HTTPError) as excinfo:
-            registered_model.create_version_from_run(
-                run_id=experiment_run.id, name="From Run {}".format(experiment_run.id)
-            )
-
-        exc_msg = str(excinfo.value).strip()
-        assert exc_msg.startswith("404")
-        assert "not found" in exc_msg
+# TODO: test_from_run_diff_workspaces with separate workspace
```

Comment:
Do you know if it is possible to keep these tests for V2?

Reply:
yes, it is possible, but we need workspace creation for that, so I added it now to workspace creation PR

Reply:
👍 Great, I see that it was added in https://github.com/VertaAI/modeldb/pull/3439/commits/29377da62da94712b41db213ebbd1612c12ae478